src/main/java/org/folio/validate/definition/AllowedFieldsConstants.java

```diff
@@ -22,7 +22,7 @@
     .addAll(COMMON_ALLOWED_FIELDS).add("checkboxField")
     .build();
   public static final Set<String> TEXT_ALLOWED_FIELDS = new ImmutableSet.Builder<String>()
-    .addAll(COMMON_ALLOWED_FIELDS).add("textField")
+    .addAll(COMMON_ALLOWED_FIELDS)
```

Comment:
Don't see the necessity to use `TEXT_ALLOWED_FIELDS` instead of `COMMON_ALLOWED_FIELDS`

Reply:
@psmagin agrre, removed